### PR TITLE
"Hello World" with Windows

### PR DESCRIPTION
**Pull request by teesloane**
# Description
A "hello world" document introducing user's to building with windows, similar to our [hello world](https://circleci.com/docs/2.0/hello-world/) document and [hello world on mac os](https://circleci.com/docs/2.0/hello-world-macos/) document.

This document runs alongside a companion [sample-app](https://github.com/CircleCI-Public/circleci-demo-windows).

### CODE REVIEW

File: jekyll/_cci2/hello-world-windows.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+* To request access to Windows Preview using [this form](https://docs.google.com/forms/d/e/1FAIpQLSfspug2MP0eTK8eRC1_FpiDQzNHkk8a36fflN_za29CwGzGoQ/viewform).
+
+
+# Overview of the Windows executor
+
+The Windows build environment (or `executor`) gives users the tools to build Windows projects, such as a Universal Windows Platform (UWP) application, a .NET executable, or Windows-specific (like the .NET framework) The Windows executor uses the Server Core version of Windows Server 2019 DataCenter Edition.
+
+A windows executor will have 4 vCPUs and 15GB of ram. Currently, there are no I/O optimizations.
+
+On the Windows executor, _Git_, the package manager _Chocolatey_, and _7zip_ will be installed. Powersell is the default shell (Bash and Command are available to be manually selected). Other dependencies may be installed later on.
```
Comment (scwheele) [inline jekyll/_cci2/hello-world-windows.md]: `Powersell is the default shell (Bash and Command are available to be manually selected).` - typo, should be Powershell

File: jekyll/_cci2/hello-world-windows.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+```
+
+Under the `jobs` key, we set the executor via the orb we are using. We can also declare the default shell to be applied across future steps in the configuration.
+
+```yaml
+    steps:
+      - checkout
+      - run:
+          name: "Install dotnet core" # https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script
+          command: .circleci/dotnet-install.ps1 -InstallDir C:\cli
```
Comment (bradylill) [inline jekyll/_cci2/hello-world-windows.md]: Feel free to do it this way, but we've fixed those pesky path issues the were breaking installations from chocolatey from working before. I've a branch on the demo project to reflect these changes. https://github.com/CircleCI-Public/circleci-demo-windows/pull/1

File: jekyll/_cci2/hello-world-windows.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+```yaml
+jobs:
+  build:
+    executor:
+      name: win/preview-default
+      shell: powershell.exe
+```
+
+Under the `jobs` key, we set the executor via the orb we are using. We can also declare the default shell to be applied across future steps in the configuration.
```
[the remark this answers is not in the document]
Comment (teesloane) [reply jekyll/_cci2/hello-world-windows.md]: Yes! Added.

File: jekyll/_cci2/hello-world-windows.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+orbs:
+  win: sandbox/windows-tools@dev:preview
+
+jobs:
+  build:
+    executor: win/preview-default
+    steps:
+      - checkout
+      - run: echo 'Hello, Windows'
```
[the remark this answers is not in the document]
Comment (teesloane) [reply jekyll/_cci2/hello-world-windows.md]: Good call, will add.

File: jekyll/_cci2/hello-world-windows.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+
+# Example configuration file
+
+The following code snippet is an absolute minimum to get started with Windows on CircleCI:
+
+```yaml
+version: 2.1
+
+orbs:
+  win: sandbox/windows-tools@dev:preview
```
[the remark this answers is not in the document]
Comment (teesloane) [reply jekyll/_cci2/hello-world-windows.md]: Done!

File: jekyll/_cci2/hello-world-windows.md (hunk 2 above)
Comment (teesloane) [reply jekyll/_cci2/hello-world-windows.md]: Ah, thank you. I was struggling with what to do to get chocolatey + dotnet happy. Thank you!

**Pull request merged**